clients/drcachesim/tracer/raw2trace.h

```diff
@@ -860,7 +867,8 @@ class raw2trace_t {
         const std::string &alt_module_dir = "",
         uint64_t chunk_instr_count = 10 * 1000 * 1000,
         const std::unordered_map<thread_id_t, std::istream *> &kthread_files_map = {},
-        const std::string &kcore_path = "", const std::string &kallsyms_path = "");
+        const std::string &kcore_path = "", const std::string &kallsyms_path = "",
+        std::istream *syscall_template_file = nullptr);
```

Comment:
Hmm... I'm only now thinking that maybe this could be a dynamorio::drmemtrace::record_reader_t instead.

Reply:
I have this as a local change. Not sure if you've already start reviewing so I'll push it later.

Reply:
Done.